storage/file.go

```diff
@@ -51,6 +51,24 @@ type FileStream struct {
 	ContentMD5 string
 }
 
+// FileRequestOptions will be passed to misc file operations.
+// Currently just Timeout but will expand.
+type FileRequestOptions struct {
+	Timeout uint
```

Comment:
I'd update this to either be a time.Duration or add documentation that states which unit this is expressed in. (i.e. Timeout is a number of seconds.)